src/renku_crac/app.py

```diff
@@ -333,11 +329,13 @@ def register_all_handlers(app: Sanic, config: Config) -> Sanic:
     """Register all handlers on the application."""
     app.add_route(ResourcePoolsView.as_view(config.rp_repo), "/api/data/resource_pools")
     app.add_route(ResourcePoolView.as_view(config.rp_repo), "/api/data/resource_pools/<resource_pool_id>")
-    app.add_route(ResourcePoolUsersView.as_view(config.user_repo), "/api/data/resource_pools/users")
-    app.add_route(ResourcePoolUserView.as_view(config.user_repo), "/api/data/resource_pools/users/<user_id>")
-    app.add_route(ClassesView.as_view(config.rp_repo), "/api/data/resource_pools/classes")
-    app.add_route(ClassView.as_view(config.rp_repo), "/api/data/resource_pools/classes/<class_id>")
-    app.add_route(QuotaView.as_view(config.rp_repo), "/api/data/resource_pools/quota")
+    app.add_route(ResourcePoolUsersView.as_view(config.user_repo), "/api/data/resource_pools/<resource_pool_id>/users")
+    app.add_route(
```

Comment:
I really find blueprints weird because they are these global variables that contain your methods. I like the way it is done with the view classes. Also it is a bit repetitive but you look at 30 lines of code and see ALL the pathing in the server. I think this is nicer than having blueprints scattered around which each contains a specific prefix.